protocols/perf/src/client/handler.rs

```diff
@@ -109,8 +109,8 @@ impl ConnectionHandler for Handler {
         event: ConnectionEvent<
             Self::InboundProtocol,
             Self::OutboundProtocol,
-            Self::InboundOpenInfo,
-            Self::OutboundOpenInfo,
+            (),
+            (),
```

Comment:
Could be omitted here and in all other handlers if we make `ConnectionEvent<_, _, IOI = (), OOI = ()>`

Reply:
Sounds interesting!

Reply:
Yes thought so too... but at least in the above code we could still avoid them, right?